plane/src/drone/executor.rs

```diff
@@ -177,8 +177,8 @@ impl Executor {
                 let callback = {
                     let state_store = self.state_store.clone();
                     let backend_id = backend_id.clone();
-                    let timestamp = chrono::Utc::now();
                     move |state: &BackendState| {
+                        let timestamp = chrono::Utc::now();
```

Comment:
Assuming this is a bug causing all events stored in the state store to have the timestamp of when this backend was created, not when the callback was called.